models/dev_staging/dev_staging_ethiopia.sql

```diff
@@ -0,0 +1,8 @@
+{{ config(
+  materialized='table'
+) }}
+
+{{ flatten_json(
```

Comment:
we can merge both the models into one using cte. we dont need this extra step

Reply:
tried doing so but we can't add cte when we use dbt_utils.star. so i took this approach